### PR TITLE
119 pathways add lesson additional boxes component

### PR DESCRIPTION
**Pull request by AdrianRotila**
119 pathways add lesson additional boxes component

### CODE REVIEW

File: src/components/AddLessonAdditionalBoxes/AddLessonAdditionalBoxes.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    height: 3rem;
+    padding: 0.75rem 1rem;
+    margin: 0.5rem 0;
+    display: inline-block;
+    box-sizing: border-box;
+    border: 1px solid #D0D5DD;
+    box-shadow: 0px 1px 2px rgba(16, 24, 40, 0.05);
+    border-radius: 0.5rem;
+    outline-color: $color-blue-navy; 
+}
```
Comment (OmarKarame) [inline src/components/AddLessonAdditionalBoxes/AddLessonAdditionalBoxes.scss:40]: Please add            &:focus {              to the .input so there won't be a blue border when clicked
                                  outline: none;
                                }

File: src/components/AddLessonAdditionalBoxes/AddLessonAdditionalBoxes.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
+@use "../../styles/variables" as *;
+
+.add-lesson {
+    display: flex;
+    flex-direction: column;
+
```
Comment (OmarKarame) [inline src/components/AddLessonAdditionalBoxes/AddLessonAdditionalBoxes.scss:6]: Please give .add-lesson a max width so it is easier to style for whoever is going to create the container.

File: src/components/AddLessonAdditionalBoxes/AddLessonAdditionalBoxes.scss [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+.input[type=text], select {
+    width: 32rem;
+    height: 3rem;
+    padding: 0.75rem 1rem;
+    margin: 0.5rem 0;
+    display: inline-block;
+    box-sizing: border-box;
+    border: 1px solid #D0D5DD;
+    box-shadow: 0px 1px 2px rgba(16, 24, 40, 0.05);
```
Comment (OmarKarame) [inline src/components/AddLessonAdditionalBoxes/AddLessonAdditionalBoxes.scss:37]: Please use variables and not rgba

File: src/components/Layout/Layout.jsx [hunk 4]
```diff
@@ -27,6 +28,8 @@ const Layout = ({ children, user }) => {
         />
       </header>
 
+      <AddLessonAdditionalBoxes />
```
Comment (OmarKarame) [inline src/components/Layout/Layout.jsx:31]: Please remove component render

**Review by andrewdohertyy — APPROVED** [review]
completed the requested changes

**Review by Connz17 — APPROVED** [review]

**Review by OmarKarame — APPROVED** [review]

**Pull request merged**